meeting-notes/2021-09-29.md

```diff
@@ -0,0 +1,63 @@
+# Meeting Notes 2021-09-29
```

Comment:
Technically this took place on the 28th right?

Reply:
Also Pro-Tip for anyone who cares/is paying attention: If you type `@today` into Google Docs, it will automatically populate with today's date

Reply:
29 in New Zealand I guess 😅

Updated. Thanks, @alicemeichi